package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "penneo-js-sdk",
   "title": "Penneo - Javascript SDK",
-  "version": "0.0.13",
+  "version": "0.0.15",
```

Comment:
It took two tries to test both approaches on npm, and versions cannot be overwritten, so `.15` syncs the version with the current npm version.